app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -122,7 +122,7 @@ class ScanSimpleIdUpdate @Inject constructor(
       model: ScanSimpleIdModel,
       identifier: Identifier
   ): ScanSimpleIdViewEffect {
-    return if (model.openedFrom == OpenedFrom.EditPatientScreen.ToAddNHID)
+    return if (model.openedFrom == OpenedFrom.EditPatientScreen.ToAddNHID || model.openedFrom == OpenedFrom.EditPatientScreen.ToAddBpPassport)
```

Comment:
We can go with this `model.openedFrom is OpenedFrom.EditPatientScreen` instead, WDYT?